_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
+questions:
+
+- "How do I activate the ESMValTool environment?"
+- "How do I configure the ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Activate the ESMValTool environment"
+- "Configure the ESMValTool"
+- "Successfully run a recipe"
+
+keypoints:
+- "See the installation guide for activating the ESMValtool environment."
+- "The config file has to be set up, uncommenting the environment you use."
```

Comment:
```suggestion
- "The information needed by ESMValTool has to be set up using `config-user.yml file`, uncommenting the Site-specific entries."
```

Reply:
Please use `environment` only for `conda environment`. You can use `site` or `infrastructure` for the machine where esmvaltool is installed.